### PR TITLE
chore: fix path exact types

### PR DESCRIPTION
**Pull request by anuraghazra**
- Adds an Exact<> type which will fix the edge cases in SVG Path props

### CODE REVIEW

File: packages/blade/src/components/Icons/_Svg/Path/Path.d.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
-export type NonDashedProps<T> = {
-  [K in keyof T]: K extends `${string}-${string}` ? never : T[K];
-};
+// https://github.com/microsoft/TypeScript/issues/12936#issuecomment-368244671
```
Comment (divyanshu013) [inline packages/blade/src/components/Icons/_Svg/Path/Path.d.ts:1]: Can add some more explanation inline here perhaps, hard to understand the linked comment 😅

Comment (anuraghazra) [reply packages/blade/src/components/Icons/_Svg/Path/Path.d.ts:1]: Will try. Kinda hard to explain that huge discussion in a few lines though.

Comment (chaitanyadeorukhkar) [reply packages/blade/src/components/Icons/_Svg/Path/Path.d.ts:1]: Just a description on what Exact does. Don't need to add all the context on why we need and all

**Review by chaitanyadeorukhkar — APPROVED** [review]
LGTM. Just add a description on what the Exact is doing

File: packages/blade/src/components/Icons/_Svg/Path/Path.d.ts (hunk 1 above)
Comment (anuraghazra) [reply packages/blade/src/components/Icons/_Svg/Path/Path.d.ts:1]: Added 

image

**Comment by anuraghazra** [comment]
Reference https://github.com/razorpay/blade/issues/648

**Pull request merged**